### PR TITLE
OFFAPPS-988 New translation Strings

### PR DESCRIPTION
**Pull request by yu-tian113**
V2 migration new translation strings

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -93,6 +93,21 @@ Refresh your page for the changes to take effect and your Sidebar Search applica
       key: "txt.apps.search_app.search.results"
       title: "number of results the placeholder here will show something like 7 results"
       value: "{{count}} result(s):"
+  - translation:
+      key: "txt.apps.search_app.search.results_count.zero"
+      title: "Label indicating the number of results, when there are 0 results."
+      screenshot: "https://drive.google.com/open?id=1XI8cFQl_LCbB1JWr72SVjv9I94Py26l1"
+      value: "{{count}} results"
```
Comment (ocke) [inline translations/en.yml]: do we need the `count` variable here? Isn't it 0?

File: translations/en.yml [hunk 2]
```diff
@@ -93,6 +93,21 @@ Refresh your page for the changes to take effect and your Sidebar Search applica
...
+  - translation:
+      key: "txt.apps.search_app.search.results_count.one"
+      title: "Label indicating the number of results, when there is 1 result."
+      screenshot: "https://drive.google.com/open?id=1_iVdHEZzojiH9Le9Pxsgi7ZHCKDxM_zk"
+      value: "{{count}} result"
```
Comment (ocke) [inline translations/en.yml]: Do we need the count variable here? Isn't it 1?

File: translations/en.yml [hunk 3]
```diff
@@ -93,6 +93,21 @@ Refresh your page for the changes to take effect and your Sidebar Search applica
...
+  - translation:
+      key: "txt.apps.search_app.search.results_count.other"
```
Comment (ocke) [inline translations/en.yml]: maybe `more` instead of `other`?

File: translations/en.yml (hunk 1 above)
Comment (vaevictis) [reply translations/en.yml]: True that. If the translation is specific to a count value, you may as well _hardcode_ the value.

**Comment by hiroiguchi** [comment]
@yu-tian113 

When you update the yml file, please cc @zendesk/i18n from next time so that we can review strings. Thanks!

**Pull request merged**